Unable to build

Since [this commit](https://github.com/shpaass/yafc-ce/commit/0528e84eb7d84d7377eeaad6f6f007d0c1be4289) I am unable to build the package on my system.

The error it generates is as following:
```
==> Starting build()...
Restore complete (0.5s)
  Yafc.I18n failed with 2 error(s) (0.2s)
    CSC : error CS2001: Source file '/home/daan/.cache/yay/factorio-yafc-ce-git/src/factorio-yafc-ce-git/Yafc.I18n/LocalizableStringClasses.g.cs' could not be found.
    CSC : error CS2001: Source file '/home/daan/.cache/yay/factorio-yafc-ce-git/src/factorio-yafc-ce-git/Yafc.I18n/LocalizableStrings.g.cs' could not be found.

Build failed with 2 error(s) in 0.8s
==> ERROR: A failure occurred in build().
    Aborting...
 -> error making: factorio-yafc-ce-git-exit status 4
```

I tried a clean build but it resulted in the same issue

> When those files are missing, run `dotnet build` without specifying a project, or explicitly run `dotnet build Yafc.I18n.Generator` to generate the files.

Don't know what's wrong here, but I got:

$ LANG=en dotnet build Yafc.I18n.Generator
Restore complete (0.3s)
  Yafc.I18n.Generator failed with 1 error(s) (0.2s) → Yafc.I18n.Generator/bin/Debug/net8.0/Yafc.I18n.Generator.dll
    /home/zehao/.cache/yay/factorio-yafc-ce-git/src/factorio-yafc-ce-git/Yafc.I18n.Generator/Yafc.I18n.Generator.csproj(11,5): error MSB3073: 
      The command "dotnet bin/Debug/net8.0//Yafc.I18n.Generator.dll
      " exited with code 150.

Build failed with 1 error(s) in 1.0s

--- a/build.sh
+++ b/build.sh
@@ -8,6 +8,7 @@ rm -rf Build
 VERSION=$(grep -oPm1 "(?<=<AssemblyVersion>)[^<]+" Yafc/Yafc.csproj)
 echo "Building YAFC version $VERSION..."
 
+dotnet build Yafc.I18n.Generator
 dotnet publish Yafc/Yafc.csproj -r win-x64 -c Release -o Build/Windows
 dotnet publish Yafc/Yafc.csproj -r win-x64 --self-contained -c Release -o Build/Windows-self-contained
 dotnet publish Yafc/Yafc.csproj -r osx-x64 -c Release -o Build/OSX

